pkg/deploymanager/config.go

```diff
@@ -0,0 +1,120 @@
+package deploymanager
+
+import (
+	"fmt"
+	"os"
+
+	olmclient "github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	lvmv1 "github.com/red-hat-storage/lvm-operator/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// InstallNamespace is the namespace lvm is installed into
+const InstallNamespace = "openshift-storage"
+
+// DeployManager is a util tool used by the tests
+type DeployManager struct {
+	olmClient      *olmclient.Clientset
```

Comment:
It might be possible to use a single controller-runtime client for everything by adding the relevant Schema .  Would be worth looking into.